test_script.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+import random
+from score import parse
+import sys
+
+# ns is the namespace. 
+# If needed write a simple parser which returns ns.
+# If simple parser should be used run the program with extra argument ('sp')
+def test(ns):
+    # TODO: Write the tests to print given ns.
+    pass
+
+def simple_parse(inp):
+    pass
+
+if __name__=='__main__':
+    args = sys.argv
+    with open(args[-1]) as f:
+        inp = f.read()
+    if len(sys.argv) > 2:
```

Comment:
Instead of doing this (which will most likely break things if we try to first use a simple parse, and later use the regular parse) can't we just send `inp` to test, and from there use `parse` or `simple_parse`? Then we don't need to switch case in the bash-script as well..